db/migrations/versions/028_create_fund_ref_table.py

```diff
@@ -0,0 +1,85 @@
+"""empty message
+
+Revision ID: 028_normalise_fund_ref_data
+Revises: 027_start_date_before_end
+Create Date: 2024-04-23 09:41:50.279834
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+import core
+
+# revision identifiers, used by Alembic.
+revision = "028_normalise_fund_ref_data"
+down_revision = "027_start_date_before_end"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "fund_dim",
+        sa.Column("fund_type_id", sa.String(), nullable=False),
+        sa.Column("id", core.db.types.GUID(), nullable=False),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_fund_dim")),
+        sa.UniqueConstraint("fund_type_id", name=op.f("uq_fund_dim_fund_type_id")),
+    )
+
+    op.execute(
+        """
+        INSERT INTO fund_dim (id, fund_type_id)
+        VALUES
+        ('4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1', 'HS'),
+        ('9fde58b2-8a89-4b2c-af7d-1f968b03c7b9', 'TD'),
+        ('e8c7c1c8-90d3-4b2d-aa50-4a2d4091d4f3', 'PF');
```

Comment:
I remember for the geospatial seeding, Noel had concerns about hard-coding the UUID that will be in the actual DBs into the migration?

Reply:
@samuelhwilliams requested above that the UUIDs be static so that they be consistent across environments. What are the concerns with hard-coding the values?

Reply:
If we've decided before to not, I'm fine to not, but be interested to hear Noel's reasons there too. It simplifies a few things in my head (eg testing), but it's not really the end of the world either way.

I've seen and used static IDs in migrations for 'reference'y data before and it's been fine.

Reply:
TBH it's probably fine, the main reasons for not doing probably don't really apply to our use case (ie not exposing DB info on public repo) so I'm happy either way, as long those keys were generated correctly and are actually unique